src/client/components/ui/Form/Select/SelectMultiple/SelectMultiple.test.tsx

```diff
@@ -0,0 +1,1007 @@
+			const user = userEvent.setup();
+
+			render(<SelectMultiple optionsAriaLabel={'options'}>
+				<SelectMultiple.Option value="1">options 1</SelectMultiple.Option>
+				<SelectMultiple.Option value="2">options 2</SelectMultiple.Option>
+			</SelectMultiple>);
+
+			await user.click(screen.getByRole('combobox'));
+			await user.click(screen.getByRole('option', { name: 'options 1' }));
+
+			expect(screen.getByRole('combobox')).toHaveTextContent('1 choix séléctionné');
+		});
+
+		it('lorsqu‘au moins deux options sont séléctionnées, je vois le nombre d‘options séléctionnées au pluriel dans le placeholder', async () => {
+			const user = userEvent.setup();
+
+			render(<SelectMultiple optionsAriaLabel={'options'}>
+				<SelectMultiple.Option value="1">options 1</SelectMultiple.Option>
+				<SelectMultiple.Option value="2">options 2</SelectMultiple.Option>
+			</SelectMultiple>);
+
+			await user.click(screen.getByRole('combobox'));
+			await user.click(screen.getByRole('option', { name: 'options 1' }));
+			await user.click(screen.getByRole('option', { name: 'options 2' }));
+
+			expect(screen.getByRole('combobox')).toHaveTextContent('2 choix séléctionnés');
+		});
+	});
+
+	describe('props', () => {
+		it('appelle onChange quand une valeur est selectionné', async () => {
+			const user = userEvent.setup();
+			const onChange = jest.fn();
+			render(<SelectMultiple optionsAriaLabel={'options'} onChange={onChange}>
+				<SelectMultiple.Option value="1">options 1</SelectMultiple.Option>
+				<SelectMultiple.Option value="2">options 2</SelectMultiple.Option>
+			</SelectMultiple>);
+
+			await user.tab();
+			await user.keyboard(KeyBoard.ENTER);
+			await user.keyboard(KeyBoard.ARROW_DOWN);
+			await user.keyboard(KeyBoard.SPACE);
+
+			expect(onChange).toHaveBeenCalledWith(expect.objectContaining({
+				tagName: 'LI',
+				textContent: 'options 2',
+			}));
+		});
+
+		it('lorsque la value change, le select prend la valeur de value mise à jour', async () => {
+			const user = userEvent.setup();
+			const valueThatCanChange = ['1'];
+			let selectValues;
+			const { rerender } = render(<form role="form">
+				<SelectMultiple optionsAriaLabel={'options'} value={valueThatCanChange}
+					name="name">
+					<SelectMultiple.Option value="1">options 1</SelectMultiple.Option>
+					<SelectMultiple.Option value="2">options 2</SelectMultiple.Option>
+				</SelectMultiple>
+			</form>);
+
+			valueThatCanChange.push('2');
+
+			rerender(<form role="form" aria-label={'test'} onSubmit={(formEvent) => {
+				selectValues = getAllFormData(formEvent, 'name');
+			}}>
+				<SelectMultiple optionsAriaLabel={'options'} value={valueThatCanChange} name="name">
+					<SelectMultiple.Option value="1">options 1</SelectMultiple.Option>
+					<SelectMultiple.Option value="2">options 2</SelectMultiple.Option>
+				</SelectMultiple>
+				<button>Submit</button>
+			</form>);
```

Comment:
Pourquoi le form prend tout un tas de nouveaux props ? 🤔 
Also, vu que c'est un tableau et que la référence ne change pas quand tu push dedans, tu devrais pouvoir variabiliser le jsx en théorie ? Un truc du genre
```suggestion
			const component = (
			    <form role="form">
				<SelectMultiple optionsAriaLabel={'options'} value={valueThatCanChange}
					name="name">
					<SelectMultiple.Option value="1">options 1</SelectMultiple.Option>
					<SelectMultiple.Option value="2">options 2</SelectMultiple.Option>
				</SelectMultiple>
			     </form>
			 );
			const { rerender } = render(component);

			valueThatCanChange.push('2');

			rerender(component);
```

Reply:
fonctionne pas à voir ensemble

Reply:
fonctionne avec la fonction